services/api-server/src/simcore_service_api_server/models/schemas/jobs.py

```diff
@@ -231,7 +218,7 @@ class JobStatus(BaseModel):
     #  SEE https://english.stackexchange.com/questions/12958/status-vs-state
 
     job_id: UUID
-    state: TaskStates
+    state: RunningState
```

Comment:
THOUGHT:  Reusing entities (i.e. models, dataclasses, etc ) from a common library (like models_library) has definitively many advantages but at least one disadvantage: a change on e.g. RunningState affects directly the public API. I wonder if there is a good strategy to control any unintended change in the public APIs.

Reply:
ok I see your point with the API. nevertheless, it was a painful red in my IDE. also there was no test checking that these were the same so I thought it was unnecessary.